www_src/components/tabs/tabs.jsx

```diff
@@ -31,7 +31,7 @@ var Tabs = React.createClass({
             'tab-body': true,
             'selected': this.state.selectedIndex === i
           });
-          return (<div className={className}>{tab.body}</div>);
+          return (<div key={i} className={className}>{tab.body}</div>);
```

Comment:
generally a bad idea. While it does stop React from complaining, there is no guarantee that the ordering for the array is stable. Is there a `tab.name` or something we can use instead so that React can do its diffing properly?